test/models/user_test.rb

```diff
@@ -4,4 +4,9 @@ class UserTest < ActiveSupport::TestCase
   # test "the truth" do
   #   assert true
   # end
+
+  test "cannot create user without a name" do
+    user = User.new
+    assert_not user.save, "blank user saved"
```

Comment:
"blank user saved" is kind of vague. Do something like "Saved user without a username"